cmake/Platform/Mac/Install_mac.cmake

```diff
@@ -6,6 +6,8 @@
 #
 #
 
+include(cmake/Platform/Common/Install_common.cmake)
```

Comment:
Does this need to include the `Install_common.cmake` script?

It seems that it would need to include `Install.cmake` instead to gain access to the install_run_code function

Reply:
No. This file is included from `Install.cmake`. We need to import `Install_common.cmake` here because it has most/all of the install functions other than the Mac specific overrides. And I don't think it's included anywhere else.